integration-tests/integration_test.go

```diff
@@ -367,6 +367,11 @@ func (s *ProcessNetworkTestSuite) TestProcessLineageInfo() {
 func (s *ProcessNetworkTestSuite) TestNetworkFlows() {
 
 	// Server side checks
+
+	// NetworkSignalHandler does not currently report endpoints
```

Comment:
Could we expand this comment (and the one below) to describe a bit more about why this is the case, and what needs to exist before we adjust this test

Reply:
Done